### ISSUE TITLE
Melgan multispaeker pretrained model fine-tuning with single speaker

### ISSUE THREAD

**Issue by MuruganR96** [descriptinc/melgan-neurips#39]
Melgan pre-trained model fine-tuning with a single speaker (720 samples - 1 hour audio samples)

it didn't work. the loss keeps increasing after the 30th epoch, and generated samples are not good

May I know any reason for this? and How can we overcome this issue?

Kindly request, Please give your suggestions.

Thanks
